libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -1378,6 +1378,18 @@ export const appStrings = {
     </UiString>
   ),
 
+  unrecoverableError: () => (
+    <UiString uiStringKey="unrecoverableError">
+      An unrecoverable error occurred.
```

Comment:
Since this is voter facing, maybe use plainer language. Could even just be "Something went wrong" since we already use that